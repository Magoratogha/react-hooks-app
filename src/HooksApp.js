function HooksApp() {
    return (
        <h1>Hello world</h1>
    );
}

export default HooksApp;