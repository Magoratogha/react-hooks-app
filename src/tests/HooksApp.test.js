import { render } from "@testing-library/react";
import HooksApp from "../HooksApp";

describe('Test for <HooksApp />', () => {
    
    test('should render the component properly', () => {
        const { container } = render(<HooksApp />);
        expect(container).toMatchSnapshot();
    });
});