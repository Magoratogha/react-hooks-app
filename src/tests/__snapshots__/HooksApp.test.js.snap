// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for <HooksApp /> should render the component properly 1`] = `
<div>
  <h1>
    Hello world
  </h1>
</div>
`;
